.devcontainer/devcontainer.json

```
@@ -1,18 +1,21 @@
 {
-    "name": "nfcore",
-    "build": {
-        "dockerfile": "Dockerfile",
-        "context": ".."
-    },
-    "remoteUser": "gitpod",
+    "name": "nextflow-training",
+
+    // Comment this and uncomment "build" if testing Dockerfile
+    // "image": "ghcr.io/nextflow-io/training:latest",
+
+    // Comment this and uncomment "image" before release
+    "build": { "dockerfile": "Dockerfile", "context": ".." },
```

When we're happy here, we should switch the comments:

```suggestion
    // Comment this and uncomment "build" if testing Dockerfile
    "image": "ghcr.io/nextflow-io/training:latest",

    // Comment this and uncomment "image" before release
    // "build": { "dockerfile": "Dockerfile", "context": ".." },
```

Once merged and released, the image will be built and pushed and codespaces will just pull it - should be significantly faster than building it each time.